openshift/templates/react-frontend/dc.yaml

```diff
@@ -128,12 +128,7 @@ objects:
           }
 
           handle /marketplace* {
-            # handle_path not working
-            uri strip_prefix /marketplace
-            reverse_proxy https://app-digmkt-${ENV}.apps.silver.devops.gov.bc.ca {
-              header_up Host {http.reverse_proxy.upstream.host}
-              flush_interval -1
-            }
+            redir https://app-digmkt-${ENV}.apps.silver.devops.gov.bc.ca
```

Comment:
please add `permanent` after the target location to create a 301, as per https://caddyserver.com/docs/caddyfile/directives/redir